packages/features/bookings/components/AvailableTimes.tsx

```diff
@@ -110,9 +110,9 @@ export const AvailableTimes = ({
 };
 
 export const AvailableTimesSkeleton = () => (
-  <div className="mt-8 flex h-full w-[20%] flex-col only:w-full">
-    {/* Random number of elements between 1 and 10. */}
-    {Array.from({ length: Math.floor(Math.random() * 10) + 1 }).map((_, i) => (
+  <div className="mt-8 flex w-[20%] flex-col only:w-full">
+    {/* Random number of elements between 1 and 6. */}
+    {Array.from({ length: Math.floor(Math.random() * 6) + 1 }).map((_, i) => (
```

Comment:
Limited the number to 6 here, so we don't get a scrollbar when there's too many skeletons.